configs/train.yaml

```diff
@@ -26,37 +28,45 @@ defaults:
   # it's optional since it doesn't need to exist and is excluded from version control
   - optional local: default.yaml
 
-  # enable color logging
-  - override hydra/hydra_logging: colorlog
-  - override hydra/job_logging: colorlog
+  # debugging config (enable through command line, e.g. `python train.py debug=default)
+  - debug: null
 
-# path to original working directory
-# hydra hijacks working directory by changing it to the new log directory
-# https://hydra.cc/docs/next/tutorials/basic/running_your_app/working_directory
-original_work_dir: ${hydra:runtime.cwd}
+# task name, determines output directory path
+task_name: "train"
 
-# path to folder with data
-data_dir: ${original_work_dir}/data/
+# default name for the experiment, determines output directory path
+# (you can overwrite this in experiment configs)
+name: "default"
 
-# pretty print config at the start of the run using Rich library
-print_config: True
+# tags to help you identify your experiments
+# (you can overwrite this in experiment configs)
+tags: ["mnist"]
```

Comment:
this tag is wrong

Reply:
Should we remove this attribute or maybe change to something like `conll2003_train` ?

Reply:
just "conll2003"

Reply:
or maybe even better: "dataset=conll2003"?

Reply:
Yeah exactly, that serves the purpose of attribute `tags`. Thanks, I'll update it.